### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by joshado**

### CODE REVIEW

**Comment by lylo** [comment]
:+1: 

Sometimes we create guidelines with the best of intentions but sometimes they do more harm than good. For the sake of no more arguments, let's allow old and new styles to coexist in love, peace and harmony and spend our time on more productive work :-)

File: README.md [hunk 1]
```diff
@@ -506,21 +506,14 @@ As you can see all the classes in a class hierarchy actually share oneclass vari
 
 - Use `Set` instead of `Array` when dealing with unique elements. `Set` implements a collection of unordered values with no duplicates. This is a hybrid of `Array`'s intuitive inter-operation facilities and `Hash`'s fast lookup.
 
-- Adopt the new hash syntax for hashes where all the keys will always be
-  symbols, for example, when creating instances using factories in tests or in
-  options hashes passed to methods.
-- For hashes where some keys may not be symbols use the old style hash rocket
-  syntax.
 - Use symbols instead of strings as hash keys where possible.
-- Don't use spaces after `{` or before `}`.
```
Comment (caius) [inline README.md]: Should this line be left in? It applies to either style of hash.

Comment (joshado) [inline README.md]: Hm, I just wondered what the intent of the rule was, given it doesn't appear to actively hurt? (cf. useless arguments around rules with no rationale).

Is there a reason?

Comment (joshado) [inline README.md]: (Also, because I tend to muscle memory a space when writing hashes)

**Pull request merged**